src/app/child-dev-project/children/child-block/child-block.component.ts

```diff
@@ -32,14 +43,28 @@ export class ChildBlockComponent implements OnInitDynamicComponent, OnChanges {
   /** prevent additional details to be displayed in a tooltip on mouse over */
   @Input() tooltipDisabled: boolean;
 
-  constructor(@Optional() private childrenService: ChildrenService) {}
+  imgPath: SafeUrl;
+  icon = Child.icon;
```

Comment:
this is rather repetitive to the child-block-tooltip implementation. But on the other hand I am no sure it's worth factoring out into its own component ... (might pay off thinking ahead toward giving any entity type the option of displaying a photo instead of icon)

Reply:
Definitely true, but seems to be out of scope for this PR as it would mostly require refactoring the tooltip component